rviz_ogre_vendor/rviz_ogre_vendorConfig.cmake.in

```diff
@@ -67,8 +70,19 @@ foreach(_lib IN LISTS OGRE_LIBRARIES)
     else()
       list(APPEND _extra_interface_link_libraries ${FREEIMAGE_LIBRARIES})
     endif()
+
     find_package(Freetype REQUIRED)
+    if(NOT FREETYPE_LIBRARIES)
+      message(FATAL_ERROR "Could not find freetype package")
+    endif()
     list(APPEND _extra_interface_link_libraries ${FREETYPE_LIBRARIES})
+
+    find_package(ZLIB REQUIRED)
+    if(NOT ZLIB_LIBRARIES)
+      message(FATAL_ERROR "Could not find zlib library")
+    endif()
+    list(APPEND _extra_interface_link_libraries ${ZLIB_LIBRARIES})
```

Comment:
We shouldn't need zlib because I disabled zip support in the ogre build config, see:

https://github.com/ros2/rviz/blob/7d939817a714ae3b6b4bfa3bf08b82c50cf440be/rviz_ogre_vendor/CMakeLists.txt#L28

If this works though, we can reenable it. Either way we should do one or the other, currently we've got the worst of both, no zip support and linking against a library we're not using.

Reply:
I tried building it again without loading zlib.

On Windows, this results in the following error:
```
freetype.lib(ftgzip.obj) : error LNK2019: unresolved external symbol inflate referenced in function FT_Gzip_Uncompress [C:\ros2\ros2_ws\build\rviz_rendering\rendering_example.vcxproj]
freetype.lib(ftgzip.obj) : error LNK2019: unresolved external symbol inflateEnd referenced in function FT_Gzip_Uncompress [C:\ros2\ros2_ws\build\rviz_rendering\rendering_example.vcxproj]
freetype.lib(ftgzip.obj) : error LNK2019: unresolved external symbol inflateReset referenced in function ft_gzip_file_io [C:\ros2\ros2_ws\build\rviz_rendering\rendering_example.vcxproj]
freetype.lib(ftgzip.obj) : error LNK2019: unresolved external symbol inflateInit2_ referenced in function FT_Gzip_Uncompress [C:\ros2\ros2_ws\build\rviz_rendering\rendering_example.vcxproj
```

Reply:
I also tried to set `-DOGRE_CONFIG_ENABLE_ZIP:BOOL` to `ON`.
This results in an error since `zzip` can not be found.

It looks like zlib is not the only library required for zip support...

Reply:
Ok, then I'll merge it as is.